src/core/Crust/Plugins/plugin_types.ts

```diff
@@ -0,0 +1,339 @@
+import { ComputedLayer, LatLngHeight, Rect } from "@reearth/core/mantle";
```

Comment:
I have defined some types depends on plugin. Thats are defined in [`Visualizer/Plugin/type.ts`](https://github.com/reearth/reearth-web/blob/ff47c54a53cdf165425a522625f255c9247abd26/src/components/molecules/Visualizer/Plugin/types.ts)